roles/docker/tasks/atomic-systemd.yml

```diff
@@ -0,0 +1,31 @@
+---
+# workaround for docker systemd unit shipped with atomic - https://bugzilla.redhat.com/show_bug.cgi?id=1318547
+# copies original systemd file from read only file system /usr -> /usr/lib/systemd/system/docker.service and removes MountFlags=slave
+# based on https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sect-Managing_Services_with_systemd-Unit_Files.html#brid-Managing_Services_with_systemd-Overriding_Unit_Mod
+
+- name: check if /etc/systemd/system/docker.service exists in atomic
+  stat:
+    path: /etc/systemd/system/docker.service
+  register: atomic_docker_systemd
+  when: ansible_os_family == "RedHat" and is_atomic_host
+
+- name: check if MountFlags=slave exists in /usr/lib/systemd/system/docker.service
+  shell: grep "MountFlags=slave" /usr/lib/systemd/system/docker.service
+  changed_when: false
+  ignore_errors: yes
+  failed_when: false
+  register: mount_flags_slave
+  when: ansible_os_family == "RedHat" and is_atomic_host and not atomic_docker_systemd.stat.exists
+
+- name: copy docker.service
+  shell: cp /usr/lib/systemd/system/docker.service /etc/systemd/system/docker.service
```

Comment:
The reason I chose to not use a templated file because I would like to use the same systemd unit file that ships with atomic. It has quite a few customizations but if we prefer a templated file would you be okay with a separate template for atomic?